libvast/src/system/connect_to_node.cpp

```diff
@@ -84,17 +84,17 @@ connect_to_node(scoped_actor& self, const caf::settings& opts) {
 #if VAST_LOG_LEVEL >= VAST_LOG_LEVEL_WARNING
   if (caf::logger::current_logger()->accepts(VAST_LOG_LEVEL_WARNING,
                                              caf::atom("vast"))) {
-    VAST_VERBOSE("{} successfully connected to {}:{}", self, node_endpoint.host,
-                 to_string(*node_endpoint.port));
+    VAST_VERBOSE("client connected to VAST node at {}:{} successfully",
```

Comment:
```suggestion
    VAST_VERBOSE("client connected to VAST node at {}:{}",
```
It either connected or failed to connect.